python/ccxt/base/exchange.py

```diff
@@ -598,6 +599,12 @@ def on_json_response(self, response_body):
         else:
             return json.loads(response_body)
 
+    def emulate_single_market_method(self, emulated_method, multi_market_method=None):
+        if multi_market_method is None:
+            multi_market_method = emulated_method + 's'
```

Comment:
I would refrain from adding this method and a string concatenation on each call, let's just keep the code in fetch_position and mark it as emulated' by default.

Reply:
> mark it as emulated' by default.

That would be be really bad, it will make it look like fetchPosition is implemented on a bunch of exchanges when it isn't

The point of the string concat was to be able to call `emulate_single_market_method` with only one argument by default 

```
this.emulate_single_market_method('fetchPosition')
```

and then the plural version would be used `fetchPositions`, but if needed you could call it with 2 arguments

```
this.emulate_single_market_method('fetchLeverageTiers', 'fetchMarketLeverageTiers')
```

I've removed the string concat part though, and now it's forced to always call it passing 2 arguments

Reply:
@kroitor The new version of this method takes the name of the single market method and the multi market method and marks the single market method as `emulated` if the multi market method is `true`